composer.json

```diff
@@ -14,7 +14,8 @@
         }
     },
     "require": {
-        "amphp/artax": "^3",
+        "php": "^7.0",
```

Comment:
Code that ran fine on `5.6` mostly runs fine on `7.0`, but I don't care enough to discuss it further. Let's wait for 8.0.